scan/tls/alert.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2009 The Go Authors. All rights reserved.
```

Comment:
No need to use a submodule in Go, you can just include from Github. For safety, the module name could be changed to scan-tls, cf-tls or something like that.